source/net/desktop-wallet/get-started-desktop.rst

```diff
@@ -11,7 +11,7 @@ The Desktop Wallet also supports storing of your private keys on a hardware devi
 Prerequisites
 =============
 
--  :ref:`Run a node <run-node-ubuntu>` on the Concordium blockchain. The node enables the Desktop Wallet to interact with the Concordium blockchain to receive updates and post transaction.
+-  Connect to (or :ref:`run a node <run-node-ubuntu>`) on the Concordium blockchain. The node enables the Desktop Wallet to interact with the Concordium blockchain to receive updates and post transaction.
```

Comment:
I do not quite understand what you intended here.

Which part confused you and should be improved? Your changes do not read quite right.

Reply:
The implication from the original text was that to use the desktop wallet you have to run a baker node locally rather than the common practice of connecting a blockchain wallet to any node, locally or externally.  Unless it's some kind of shadow policy, it seemed like bad engrish to me.

Reply:
I see. I think the suggested phrasing implies some dichotomy between connecting and running.
The node also has to be trusted.
```suggestion
-  Connect to a **trusted** node on the Concordium blockchain. Either :ref:`your own <run-node-ubuntu>` or a third-party node. The node enables the Desktop Wallet to interact with the Concordium blockchain to receive updates and post transaction.
```

Reply:
There is also incorrect punctuation in this and disagreement between an object and verb. See below for corrected version of Ales's suggestion.

Connect to a **trusted** node on the Concordium blockchain, either :ref:`your own <run-node-ubuntu>` or a third-party node. The node enables the Desktop Wallet to interact with the Concordium blockchain to receive updates and post transactions.